int8[] num
string[] class_name
int8[] conf
int16[] box_x
int16[] box_y
int16[] size_x
int16[] size_y
